### PR TITLE
[WEB-436] Clean up at a glance box

### PR DESCRIPTION
**Pull request by rhyolight**
screen shot 2017-01-10 at 3 25 50 pm

### CODE REVIEW

**Comment by rhyolight** [comment]
@lscheinkman or @brev please review

**Comment by lscheinkman** [comment]
@rhyolight I got this warning in the console:
```
Warning: Failed prop type: The prop `to` is marked as required in `TextLink`, but its value is `undefined`.
    in TextLink (created by SectionHome)
    in SectionHome
    in div (created by Section)
    in section (created by Section)
    in Section (created by HomePage)
    in div (created by HomePage)
    in HomePage (created by RouterContext)
    in main (created by Main)
    in Main (created by Layout)
    in div (created by Layout)
    in Layout (created by Template)
    in Template (created by RouterContext)
    in RouterContext (created by Router)
    in ScrollBehaviorContainer (created by Router)
    in Router
```

File: packages/numenta.org/pages/_Section.jsx [hunk 1]
```diff
@@ -146,7 +157,7 @@ const SectionHome = (props, {config}) => {
             Github organization
           </TextLink> {' '}
           . If you're new to HTM, you'll learn a lot in {' '}
-          <TextLink to={links.out.school}>
+          <TextLink to={links.in.school}>
```
Comment (lscheinkman) [inline packages/numenta.org/pages/_Section.jsx]: It should be `links.out.school`

Comment (rhyolight) [inline packages/numenta.org/pages/_Section.jsx]: Thank you, I missed that one.

**Comment by rhyolight** [comment]
@lscheinkman I've seen that browser error before, but I'm not getting it currently on this branch.

**Comment by lscheinkman** [comment]
👍 It works

**Comment by lscheinkman** [comment]
The browser error was fixed by c3b58e2 . It was related to the `links.out.school`
👍  It all works now!

**Pull request merged**